README.md

```diff
@@ -1,6 +1,6 @@
-# Icon Data Processing Incubator: Prototype for a data post-processing framework on the basis of xarray.
+# meteodata-lab: Prototype for a model data processing framework on the basis of xarray.
```

Comment:
Do we want to keep the word prototype?

Reply:
We could remove it from the title but add a section below as in the [ecmwf/polytope](https://github.com/ecmwf/polytope/blob/develop/readme.md) repo:

```
> [!WARNING]
> This project is BETA and will be experimental for the forseable future. Interfaces and functionality are likely to change, and the project itself may be scrapped. DO NOT use this software in any project/software that is operational.
```